docs/reference/embedding.rst

```diff
@@ -6,6 +6,27 @@ Embedding
 
 Provides functions that map binary quadratic models and samples between a source graph and a target graph.
 
+The D-Wave system is Chimera-structured. The Chimera architecture comprises sets of
```

Comment:
The `dwave.embedding` namespace is more-or-less graph agnostic (with the obvious exception of the `chimera` subpackage) so I wonder if we shouldn't reference Chimera at at this level? On the other hand I see the expedience of doing so.

Maybe for some future proofing we can say "the D-Wave 2000Q is Chimera-structured".

Reply:
As I worked though the repo, I was continually tempted to remove references to Chimera but I prefer to update everything for new architecture as one comprehensive task.